packages/legend-application-marketplace/src/components/Lakehouse/entitlements/LakehouseEntitlements.tsx

```diff
@@ -307,6 +475,18 @@ export const LakehouseEntitlements = withLakehouseEntitlementsStore(
       );
     }, [auth.user?.access_token, entitlementsStore, params]);
 
+    const renderCurrentViewer = (): React.ReactNode => {
+      const currentViewer = entitlementsStore.currentViewer;
+      if (currentViewer instanceof LakehousePendingTasksViewer) {
+        return <LakehouseEntitlementsTasks currentViewer={currentViewer} />;
+      } else if (currentViewer instanceof DataContractTaskState) {
+        return <LakehouseEntitlementsTask currentViewer={currentViewer} />;
```

Comment:
Any reason for not just doing
```suggestion
      if (currentViewer instanceof LakehousePendingTasksViewer || currentViewer instanceof DataContractTaskState) {
        return <LakehouseEntitlementsTasks currentViewer={currentViewer} />;
      }
```

Reply:
one is `LakehouseEntitlementsTask**s**` the other is just `LakehouseEntitlementsTask`

Reply:
Ah my bad, missed the difference.